samples/dotnet/kernel-syntax-examples/Example17_ChatGPT.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.AI.ChatMessageGeneration;
+using Microsoft.SemanticKernel.Configuration;
+using RepoUtils;
+
+/**
+ * The following example shows how to use Semantic Kernel with OpenAI ChatGPT API
+ */
+
+// ReSharper disable once InconsistentNaming
+// ReSharper disable CommentTypo
+public static class Example17_ChatGPT
+{
+    public static async Task RunAsync()
+    {
+        Console.WriteLine("======== SK with ChatGPT ========");
+
+        IKernel kernel = new KernelBuilder().WithLogger(ConsoleLogger.Log).Build();
+
+        // Add your chat completion backend
+        kernel.Config.AddOpenAIChatCompletion("chat", "gpt-3.5-turbo", Env.Var("OPENAI_API_KEY"));
+
+        IChatCompletion backend = kernel.GetService<IChatCompletion>();
+        var chat = (OpenAIChatHistory)backend.CreateNewChat("You are a librarian, expert about books");
```

Comment:
It'd be really cool if we could take existing TextCompletion prompts and import them as ChatGPT prompts (auto pull the `description` into the System messages, etc.) -- did we consider this at all?

Reply:
yes once we have this in we can start chaining semantic functions into system messages. We need to find a compelling scenario/example. It think the Q&A app would be a good playground to try